src/seedu/addressbook/commands/SortCommand.java

```diff
@@ -0,0 +1,23 @@
+package seedu.addressbook.commands;
+
+
+/**
+ * Sorts the address book.
+ */
+public class SortCommand extends Command {
+
+    public static final String COMMAND_WORD = "sort";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Sorts the address book by name."
+                                               + "Example: " + COMMAND_WORD;
+
+    public static final String MESSAGE_SUCCESS = "Addressbook sorted!";
```

Comment:
Hey Jethro, would it be better if this is private instead, since it is only used in this class.